src/test/java/sirius/kernel/commons/LimitTest.java

```diff
@@ -86,4 +85,22 @@ public void testPredicate() {
         int sum = Stream.of(1, 2, 3, 4, 5, 6, 7, 8, 9, 10).filter(new Limit(2, 4).asPredicate()).mapToInt(i -> i).sum();
         assertEquals(3 + 4 + 5 + 6, sum);
     }
+
+    @Test
+    public void testRemaingItems() {
+        Limit l = new Limit(5, 500);
+        assertEquals(Integer.valueOf(505), l.getRemainingItems());
+        l = new Limit(10, 0);
+        assertNull(l.getRemainingItems());
```

Comment:
shouldn't this be 10?

Reply:
Limits created with maxItems < 1 are unlimited